### DISCUSSION TITLE
部署报错，Error: Notion page not found "Febxxxxxxx"，notionid含有中间横线出错

### DISCUSSION THREAD

**Discussion by muyudou** [tangly1024/NotionNext#2016] [Q&A]
**描述bug**
【此项必填】修改了notionId之后报错，id的格式是Feb-xxxxxxxxxxxxxx
<img width="641" alt="image" src="https://github.com/tangly1024/NotionNext/assets/5087462/ac93a9b6-f994-4139-a5e0-cad1ae69c557">
<img width="1035" alt="image" src="https://github.com/tangly1024/NotionNext/assets/5087462/afa5cec3-d98b-4fb7-b4fa-3d7fdd1bb999">

**复现步骤**
【此项必填】出现这个bug的操作步骤

**期望的正常结果**
【此项必填】希望按这个步骤，正常操作结果是什么

**截图**
【可选】相关的页面，应该的结果

<img width="504" alt="image" src="https://github.com/tangly1024/NotionNext/assets/5087462/c9f12ee1-01d6-4c40-aaf0-4e5e10fa32e8">

**环境**

- 【必填】NotionNext版本 [例如. 4.2.4]
- 【必填】主题 notion模版
- 【必填】部署方案 vercel
- 【可选】操作系统: macOS
- 【可选】浏览器 chrome

**补充说明**
【可选】与问题相关的其它说明

**Comment by tangly1024** [comment]
请尝试调整id格式，

`Feb-xxxxxxxxxxxxxx` 改为 `xxxxxxxxxxxxxxxxx`

即： 去掉`Feb-`
